### PR TITLE
Added security review tooling

### PR DESCRIPTION
**Pull request by lumjjb**
Addressing https://github.com/cncf/sig-security/issues/335

### CODE REVIEW

File: assessments/guide/tools.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+This document contains a set of tools that will be useful to security reviewers and project leads.
+
+## Document conversion
+
+Self-assessments are sometimes done in a Google doc, which facilitates discussions. At the end of the
+review, the documents needs to be converted to markdown so that they can be added to the github repository.
+
+List of tools:
+- [Pandoc](https://pandoc.org/)
```
Comment (ultrasaurus) [inline assessments/guide/tools.md]:
```suggestion
- [Pandoc](https://pandoc.org/)
- [Google Docs extension: docs-to-markdown](https://chrome.google.com/webstore/detail/docs-to-markdown/igffnbdfnodiaphfmfaiiaegmoljbghf)
```

**Comment by lumjjb** [comment]
Hmm so maybe like `/logistics/tools.md`?

File: assessments/guide/tools.md (hunk 1 above)
Comment (SantiagoTorres) [reply assessments/guide/tools.md]: I used [grip](https://github.com/joeyespo/grip), if I recall correctly, although I don't quite remember what I used for gdocs -> markdown (maybe some sed/awk or [docx2txt](https://github.com/ankushshah89/python-docx2txt) + manual edits)

**Comment by ultrasaurus** [comment]
I like the idea of having a `/logistics` folder -- that can totally happen as a refactor.  Let's go ahead and merge so this list can be avail for folks!

**Pull request merged**